### PR TITLE
Fix scaling issue

### PR DESCRIPTION
**Pull request by chr11115**
- Scale geometry such that it uses proper units (prt uses meter, while maya uses centimeter -> scale difference of 100)

### CODE REVIEW

**Review by mistafunk — CHANGES_REQUESTED** [review]
(deleted wrong review comment)

**Review by mistafunk — CHANGES_REQUESTED** [review]
LGTM, but let's do the scaling in double precision to be on the safe side for big/small coordinates.

File: src/serlio/modifiers/MayaCallbacks.cpp [hunk 1]
```diff
@@ -64,7 +64,7 @@ MFloatPointArray toMayaFloatPointArray(double const* a, size_t s) {
 	MFloatPointArray mfpa(numPoints);
 	for (unsigned int i = 0; i < numPoints; ++i) {
 		mfpa.set(MFloatPoint(static_cast<float>(a[i * 3 + 0]), static_cast<float>(a[i * 3 + 1]),
-		                     static_cast<float>(a[i * 3 + 2])),
+		                     static_cast<float>(a[i * 3 + 2])) * mu::PRT_TO_SERLIO_SCALE,
```
[the remark this answers is not in the document]
Comment (chr11115) [reply src/serlio/modifiers/MayaCallbacks.cpp]: fixed, using double precision now

**Review by mistafunk — APPROVED** [review]
LGTM, feel free to merge.

**Pull request merged**